tests/e2e/e2e_test.go

```diff
@@ -21,6 +21,22 @@ import (
 var curProj string
 var testNamespacedImage = "https://raw.githubusercontent.com/bucharest-gold/centos7-s2i-nodejs/master/imagestreams/nodejs-centos7.json"
 
+// EnvVarTest checks the component container env vars in the build config for git and deployment config for git/binary/local
+// appTestName is the app of the app
+// sourceType is the type of the source of the component i.e git/binary/local
+func EnvVarTest(resourceName string, sourceType string) {
+
+	if sourceType == "git" {
+		// checking the values of the env vars pairs in bc
+		envVars := runCmd("oc get bc " + resourceName + " -o go-template='{{range .spec.strategy.sourceStrategy.env}}{{.name}}{{.value}}{{end}}'")
+		Expect(envVars).To(Equal("keyvaluekey1value1"))
+	}
+
+	// checking the values of the env vars pairs in dc
+	envVars := runCmd("oc get dc " + resourceName + " -o go-template='{{range .spec.template.spec.containers}}{{range .env}}{{.name}}{{.value}}{{end}}{{end}}'")
+	Expect(envVars).To(Equal("keyvaluekey1value1"))
```

Comment:
This shouldn't be hard-coded right? Shouldn't we pass these "test" variables with parameters?